src/io_uring/tests/integration_tests.rs

```diff
@@ -22,12 +24,18 @@ fn test_ring_new() {
 
     // Invalid entries count: 0.
     assert!(matches!(
-        IoUring::new(0),
+        IoUring::new(0, true),
         Err(Error::Setup(e)) if e.kind() == std::io::ErrorKind::InvalidInput
     ));
 
     // Valid entries count.
-    assert!(IoUring::new(NUM_ENTRIES).is_ok());
+    assert!(IoUring::new(NUM_ENTRIES, true).is_ok());
+
+    // Disabled.
+    assert!(!IoUring::new(NUM_ENTRIES, false).unwrap().enabled());
+
+    // Enabled.
+    assert!(IoUring::new(NUM_ENTRIES, true).unwrap().enabled());
```

Comment:
Doesn't this line also cover the `assert!(IoUring::new(NUM_ENTRIES, true).is_ok());` case tested above?

Reply:
removed this